Sources/CartonCLI/Commands/Test.swift

```diff
@@ -41,6 +45,12 @@ struct Test: AsyncParsableCommand {
   )
   private var environment = Environment.wasmer
 
+  // It implemented as a separate flag instead of a `--environment` variant because `--environment`
+  // is designed to accept specific browser names in the future like `--environment firefox`. 
+  // Then `--headless` should be able to be used with `defaultBrowser` and other browser values.
```

Comment:
Sorry, I wanted to suggest a doc comment, but used a regular comment delimiter for the previous suggestion.
```suggestion
  /// It is implemented as a separate flag instead of a `--environment` variant because `--environment`
  /// is designed to accept specific browser names in the future like `--environment firefox`. 
  /// Then `--headless` should be able to be used with `defaultBrowser` and other browser values.
```